dynawo/3rdParty/CMakeLists.txt

```diff
@@ -508,7 +526,14 @@ else()
 
       INSTALL_COMMAND     ""
     )
+
   else()
+    if(CXX11_ENABLED)
+      set(TmpVar "--with-libraries=date_time,filesystem,program_options,serialization,system,log,iostreams,unit_test_framework")
```

Comment:
unit_test_framework name is only test in this option

Reply:
on this line yes

Reply:
I found it... done